apps/meteor/client/views/marketplace/helpers/handleAPIError.ts

```diff
@@ -8,10 +8,10 @@ const shouldHandleErrorAsWarning = (message: string): boolean => {
 };
 
 export const handleAPIError = (errorObject: unknown): void => {
-	const { message = '', error = '' } = errorObject as { message?: string; error?: string };
+	const { error = '', message = error } = errorObject as { message?: string; error?: string };
```

Comment:
[visible_confusion] Why not use only error as message then?

Reply:
[visible_confusion_as_well] The `errorObject` might contain `message`, or `error`, or `both`. In case it has both, prefer `error`

Reply:
Oh, I see... it's optional